#ifndef MALWAREINCIDENT_H
#define MALWAREINCIDENT_H

#include <QDialog>

namespace Ui {
class malwareincident;
}

class malwareincident : public QDialog
{
    Q_OBJECT

public:
    explicit malwareincident(QWidget *parent = nullptr);
    ~malwareincident();
    QString generateRandomID() const;
    void saveIncident();

private slots:
    void on_malwareSave_clicked();

private:
    Ui::malwareincident *ui;
};

#endif // MALWAREINCIDENT_H
